src/transcoder/transcoderoptionsflac.cpp

```diff
@@ -37,9 +39,30 @@ void TranscoderOptionsFlac::Load() {
   ui_->quality->setValue(s.value("quality", 5).toInt());
 }
 
+void TranscoderOptionsFlac::ValueChanged(int value) {
+  if (!IsInStreamingSubset(value)) {
+    ui_->info->setStyleSheet("QLabel { color : red }");
+    ui_->info->setText(
+        tr("Warning: This compression level is outside of the streamable "
+           "subset."));
+  } else {
+    ui_->info->setText("");
+  }
+}
+
 void TranscoderOptionsFlac::Save() {
   QSettings s;
   s.beginGroup(kSettingsGroup + settings_postfix_);
 
-  s.setValue("quality", ui_->quality->value());
+  int quality = ui_->quality->value();
+  s.setValue("quality", quality);
+
+  s.setValue("streamable-subset", IsInStreamingSubset(quality));
+}
+
+bool TranscoderOptionsFlac::IsInStreamingSubset(int level) {
+  // The gstreamer flacenc element defines its own quality settings. The
```

Comment:
A link to the FLAC docs would be useful here.

Reply:
Done.